x-pack/plugins/lens/public/vis_type_alias.ts

```diff
@@ -12,19 +12,12 @@ export const getLensAliasConfig = (): VisTypeAlias => ({
   aliasPath: getBasePath(),
   aliasApp: 'lens',
   name: 'lens',
-  promotion: {
-    description: i18n.translate('xpack.lens.visTypeAlias.promotion.description', {
-      defaultMessage: 'Try Lens, our new, intuitive way to create visualizations.',
-    }),
-    buttonText: i18n.translate('xpack.lens.visTypeAlias.promotion.buttonText', {
-      defaultMessage: 'Go to Lens',
-    }),
-  },
+  promotion: true,
   title: i18n.translate('xpack.lens.visTypeAlias.title', {
     defaultMessage: 'Lens',
   }),
   description: i18n.translate('xpack.lens.visTypeAlias.description', {
-    defaultMessage: `Lens is a simpler way to create basic visualizations`,
+    defaultMessage: `Drag and drop intuitive way to create multiple types of visualizations. Best option for most visualizations.`,
```

Comment:
This sentence had some confusing grammar, I would suggest this:

```suggestion
    defaultMessage: `Drag and drop to create multiple types of visualizations. Best option for most visualizations.`,
```

cc @gchaps

Reply:
Thank you @wylieconlon, texts are not final.

Reply:
Changed it with Gail's suggestion.